index.js

```diff
@@ -1,88 +1,114 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
+counter = 1
+counter = 2
 
```

Comment:
While this works, it's best practice to declare variables with `let` or `const`. In this case, we would need to use `let` because `let` allows us to reassign our variables